astroquery/mast/core.py

```diff
@@ -702,7 +702,7 @@ def _session_info(self, silent=False):  # pragma: no cover
         """
 
         # get user information
-        response = self._session.request("GET", self._SESSION_INFO_URL)
+        response = self._session.request("GET", self._SESSION_INFO_URL, headers={"Accept": "application/json"})
```

Comment:
Suggest you just update the session headers instead of overwriting them:
`self._session.headers["Accept"] = "application/json"`

Or else build a header object that includes what's already in `self._session.headers`

Reply:
This brings up an interesting point.  At one point this code worked.  I wonder when that header was removed from the global scope.

Does overriding the headers param override the session's headers?

Reply:
If you set them in the Object._session.headers I believe so.  It's definitely possible that that is what happened.